### ISSUE TITLE
Election process : remove prohibition to abstain?

### ISSUE THREAD

**Issue by jeanlucchristin** [holacracyone/Holacracy-Constitution#295]
I propose to simply withdraw the prohibition to abstain during integrative election process. 

Concretely, remove the three words that I put here between parentess

**3.3.6 Integrative Election Process**
 (b) Fill Out Ballots: Each participant **(must)** then fills out a ballot to nominate whomever the participant believes is the best fit for the Role among all eligible candidates. Each participant **(must)** labels the ballot with his or her own name as well, and no one may **(abstain or)** nominate multiple people. Before and during this step, the Facilitator must promptly stop all comments or discussion about potential candidates or nominations.

**underlying tension :**
A participant left an election meeting because he could not abstain. He expressed that he did not understand the meaning of such a constraint and questioned the adoption of holacracy.
I wondered what interest this ban had.

after having exchanged on two forums (holacracy and a french one), I found no one who forbids the abstention and nobody who can explain the interest to prohibit it.
Having a constitution that unambiguously matches our practice would bring clarity

**Comment by matthiask** [comment]
Even though we aren't working with Holacracy for a long time yet it seems obvious to me that having to cast a vote leads to a broader distribution of power

Of course abstaining from a vote would sometimes be easier, especially for the silent majority (minority?). Allowing abstention leaves more power with those who already had it before adopting Holacracy. I personally think that allowing this would be a bad idea.

Also, the votes by those who asked whether abstention was an option were interesting and -- while maybe unexpected -- certainly good choices.

**Comment by klaas1979** [comment]
I would +1 mathiask:
I am facilitating Governance meetings as internal coach for 2 years. If it is a question why you have to nominate anyone, I always say: we do not search for the best person for the role but one that works for the whole circle. For this we need the data. AND it is very nice to get a warm shower of reasons, why somebody is nominated.

I would fear hidden power struggles if people get the option to abstain. It is a lot more risky to nominate if abstain is an option and a lot of people do this.

If someone leaves the meeting I would deal with this single tension without changing the rules. Mayte this is more a personal tension or a tension in the community space that should be dealt with in other spaces?

**Comment by jeanlucchristin** [comment]
Thank you for these enlightening comments.

I realize that I am left with a questioning after some days : Is it not a paradoxical injunction to want to force someone to take responsibility?
You know : something that sounds like "I order you to be free"

I posted this too on holacracy.org/forum, I copy this reflection there

**Comment by brianjrobertson** [comment]
I think Chris and others hit on the value of this rule in [this thread](https://community.holacracy.org/topic/why-impose-everyone-to-vote-during-intergative-election?reply=590444325828037854#590444325828037854).  We'd lose that value if we just dropped this rule; what could be done instead to solve this tension without losing this value?

**Comment by jeanlucchristin** [comment]
I answer to @brianjrobertson "what could be done instead to solve this tension without losing this value?"

With Chris's , I see that the source of his thought is in "**I suspect** that it's going to be in the likelihood that voting will become politicized." 
It states "you do not want someone abstaining because **they're afraid** to make their opinion transparent, so voting isn't a choice."
That's a guess, you do not know what's in their head or why they want to abstain

Chris doesn't "feel super-confident in this answer because he has never practiced the process without requiring everyone to vote"

So the value you mention @brianjrobertson is that of **ensuring that voting will not become politicized.** 
I think we don't need that value

**For my part, I do not want to be fear driven. 
I want to be purpose and process driven**

I've no fear with the politization because of the election process. it is the framework that protects from the effect of politicization.
If one abstains and lets the group decide for him, for example because he has no tension with the election of one or the other, it does not frighten me. If later he has an objection, the process will test it and if necessary, will integrate it.

I saw that you rejected the associated pull request before the closing of this discussion, which means maybe the decision is obvious to you and already taken.
In this case I understand your position but I am not in the same project.
Thank you for this exchange

**Comment by MiekeByerley** [comment]
My very first question would be. What is the motivation for the Abstaining? I think that for me is a much more important aspect to understand. The problem is that many think that voting is a priviledge function to voice ones own opinion but this is a misconception, Voting is an obligation you hold towards the collective and organization for the betterment of the whole. In other words regardless of your interest in the outcome of a vote personally, you have an obligation as a member of an organization to ensure that the outcome is to the greatest benefit of the whole.

**Comment by jeanlucchristin** [comment]
@MiekeByerley 
I like this idea of questioning everyone about the meaning of voting.
I also like to remind everyone that their choices as a member must be made for the benefit of their roles and even for the benefit of the collective (although the latter concept does not appear in the constitution).

There remains the initial question: my preference is that everyone vote, does it allow me to make it a constitutional obligation?
For what danger should we be able to force someone to act as we want him to act?

**Comment by brianjrobertson** [comment]
@jeanlucchristin I closed the pull request because the discussion is still ongoing and my process is to reject them until the issue is clearly accepted; it is neither accepted nor rejected yet.

**Comment by brianjrobertson** [comment]
My current take:  This is not a matter of "forcing someone to act", no more than the requirement to process tensions for your role, or track your projects, or anything else required in the constitution.  Typically, someone joining a Holacracy-powered organization is explicitly asked to agree to enact the rules in the constitution, including the one that they will vote during elections.  So it's simply that - another agreement someone makes to participate in a Holacracy-powered company.  Thus, I think the real question here is not "should we force people to do this", but rather, "should this be a required agreement for Holacracy practice"?  My answer to that is currently yes, for the reasons Chris articulated in his forum post.  Thoughts?

**Comment by jeanlucchristin** [comment]
Hello @brian

I understand your point of view and I think you have perfectly refocused the question : "should this be a required agreement for Holacracy practice?"
 
I'm puzzled by the fact that you rely on Chris's post.
This post does not give arguments based on reality but an argument based on suspicions.

you ask for thoughts. My contribution is to ask you: can you rephrase what tangible argument you retain here?

**Comment by brianjrobertson** [comment]
I found Chris' experience and the arguments he makes from it really convincing, and he's one of the most expert Holacracy coaches in the world in my view; if I were to repeat my argument and sense of it, it would pretty much be the same that he's already shared - I don't think I have anything to add, and nor do I see an issue with the way he's drawn on his experience to make the arguments (I'm not getting your point about why that would be a problem in this case/context).

**Comment by brianjrobertson** [comment]
For reference, here's [the link](https://community-archive.holacracy.org/topic/why-impose-everyone-to-vote-during-intergative-election?reply=590444325828037854#590444325828037854) to Chris' post I'm referencing (the link shared earlier in this thread no longer works, as a result of the move to the new Community of Practice platform; the link given here goes to the post in the archived old Community of Practice).

**Comment by brianjrobertson** [comment]
Given the lack of additional comments over this past week, and my own perspective that it should indeed be a required agreement of all partners to avoid abstaining in elections, I'm going to go ahead and close this issue; that said, if anyone has further thoughts or debate, please comment regardless and I'll open it up again if needed.
